app/domain/errors.rb

```diff
@@ -273,6 +273,16 @@ module AuthnK8s
           msg:  "Invalid Kubernetes host id: {0}. Must end with <namespace>/<resource_type>/<resource_id>",
           code: "CONJ00048E"
         )
+
+        CSRMissingCNEntry = ::Util::TrackableErrorClass.new(
+          msg:  "CSR [subject: '{0-subject}', spiffe_id: '{1-spiffe-id}'] must have a CN entry.",
```

Comment:
@shulifink please review this

Reply:
fine

Reply:
do they know what a CN entry is? Actually Jason mention the other day that we should never abbreviate CN and SON.

Reply:
CN is common name.
san is Subject Alternate Name.

Reply:
I would still call it a `CN entry` because the certificate will actually have the word `CN` in it. However, we can say `CN (Common-Name) entry`.

Reply:
.... must have a CN (common name) entry